projects/swimlane/ngx-ui/src/lib/components/date-range-calendar/date-range-picker.component.html

```diff
@@ -83,8 +88,22 @@
       </div>
       <div class="right">
         <ngx-button class="btn btn-bordered" (click)="onCancel()">Cancel</ngx-button>
-        <ngx-button class="btn btn-primary" [disabled]="!!validationError" (click)="onApply()"> Apply </ngx-button>
+        <ngx-button class="btn btn-primary" [disabled]="isApplyDisabled" (click)="onApply()"> Apply </ngx-button>
       </div>
     </div>
   </ngx-dropdown-menu>
 </ngx-dropdown>
+
+<ng-template #startdefaultTooltipTemplate>
+  <div *ngFor="let item of timeValueStart | keyvalue" class="date-zone-display">
```

Comment:
Please change this to control flow.
Do not combine structural directives (*ngFor, *ngIf, etc) with Control Flow (@if, @for)
Structural directives are going to be deprecated in the short-term by the angular team